src/plugins/data/common/field_formats/converters/duration.ts

```diff
@@ -100,50 +100,80 @@ const outputFormats = [
     text: i18n.translate('data.fieldFormats.duration.outputFormats.asMilliseconds', {
       defaultMessage: 'Milliseconds',
     }),
+    shortText: i18n.translate('data.fieldFormats.duration.outputFormats.asMilliseconds.short', {
+      defaultMessage: 'ms',
+    }),
     method: 'asMilliseconds',
   },
   {
     text: i18n.translate('data.fieldFormats.duration.outputFormats.asSeconds', {
       defaultMessage: 'Seconds',
     }),
+    shortText: i18n.translate('data.fieldFormats.duration.outputFormats.asSeconds.short', {
+      defaultMessage: 'sec',
```

Comment:
Come to think of it, I think we might be using `s` instead of `sec` (at least in APM). I think both are fine, I just prefer `s` because it's shorter and will fit better in more condensed usages, like tables and charts.